src/Components/Facility/FacilityHome.tsx

```diff
@@ -80,76 +107,158 @@ export const FacilityHome = (props: any) => {
+          routes.listDoctor,
+          {
+            pathParams: { facilityId: facilityId },
+          }
+        );
+        const { res: triageRes, data: triageData } = await request(
+          routes.getTriage,
+          {
+            pathParams: {
+              facilityId: String(facilityId),
+            },
+          }
+        );
+
+        if (permittedFacilityRes) {
           setIsLoading(false);
-          if (!facilityRes.data) {
+          if (!permittedFacilityData) {
             Notification.Error({
               msg: "Something went wrong..!",
             });
           } else {
-            setFacilityData(facilityRes.data);
-            if (capacityRes && capacityRes.data) {
-              setCapacityData(capacityRes.data.results);
+            const transformedPermittedFacilityData = {
+              id: Number(permittedFacilityData.id),
+              name: permittedFacilityData.name,
+              district: permittedFacilityData.district,
+              read_cover_image_url: permittedFacilityData.read_cover_image_url,
+              facility_type: String(permittedFacilityData.facility_type),
+              address: permittedFacilityData.address,
+              features: permittedFacilityData.features,
+              location: {
+                latitude: Number(permittedFacilityData.latitude),
+                longitude: Number(permittedFacilityData.longitude),
+              },
+              oxygen_capacity: permittedFacilityData.oxygen_capacity,
+              phone_number: permittedFacilityData.phone_number,
+              type_b_cylinders: permittedFacilityData.type_b_cylinders,
+              type_c_cylinders: permittedFacilityData.type_c_cylinders,
+              type_d_cylinders: permittedFacilityData.type_d_cylinders,
+              middleware_address: permittedFacilityData.middleware_address,
+              expected_type_b_cylinders:
+                permittedFacilityData.expected_type_b_cylinders,
+              expected_type_c_cylinders:
+                permittedFacilityData.expected_type_c_cylinders,
+              expected_type_d_cylinders:
+                permittedFacilityData.expected_type_d_cylinders,
+              expected_oxygen_requirement:
+                permittedFacilityData.expected_oxygen_requirement,
+              local_body_object: permittedFacilityData.local_body_object,
+              district_object: permittedFacilityData.district_object,
+              state_object: permittedFacilityData.state_object,
+              ward_object: permittedFacilityData.ward_object,
+              modified_date: permittedFacilityData.modified_date,
+              created_date: permittedFacilityData.created_date,
+              state: permittedFacilityData.state,
+              local_body: permittedFacilityData.local_body,
+              ward: permittedFacilityData.ward,
+            };
+            setFacilityData(transformedPermittedFacilityData);
```

Comment:
Maintaining large chunks of code like this is super hard. If a new field is added or renamed or field is removed, it's difficult to update it in lots of places.

```suggestion
            setFacilityData(permittedFacilityData);
```